src/shims/unix/linux/mod.rs

```diff
@@ -1,3 +1,4 @@
 pub mod dlsym;
 pub mod foreign_items;
+pub mod fs;
```

Comment:
None of the stuff in there is really about the "file system"... maybe rename this to `fd`? It is about file descriptors, after all. :shrug: